java/java-dataflow-samples/read-pubsub-write-bigquery/README.md

```diff
@@ -66,8 +66,109 @@ Start your journey at the main method here:
 
 # Getting Started with IntelliJ
 
-TODO
+The following steps guide you how to use this sample.
+
+## Provision required resources
+
+[![Open in Cloud Shell](https://gstatic.com/cloudssh/images/open-btn.svg)](https://shell.cloud.google.com/cloudshell/editor?cloudshell_git_repo=https%3A%2F%2Fgithub.com%2FGoogleCloudPlatform%2Fcloud-code-samples&cloudshell_git_branch=v1&cloudshell_tutorial=cloud-shell-readme.md&cloudshell_working_dir=java%2Fjava-dataflow-samples%2Fread-pubsub-write-bigquery&show=terminal)
+
+## Acquire gradle.properties file
+
+The previous provision required resources step will generate this for you and
+download to your machine.  Move this downloaded `gradle.properties` file to your
+IDE working directory on your local machine.
+
+If you lost it, you can download it again from Google Cloud Storage:
+1. Find the Google Cloud storage bucket containing the `gradle.properties` file.
+Cloud Code's Google Cloud Storage allows you to list and view buckets and
+objects.  Just locate the `gradle.properties` file and copy the bucket name.
+![copy_bucket_name_to_clipboard](img/copy_bucket_name_to_clipboard.png)
+2. [Open the IntelliJ terminal](https://www.jetbrains.com/help/idea/terminal-emulator.html)
+and run the following command.  NOTE: `<bucket>` is the name of the bucket
+you copied in the previous step.
+```sh
+gsutil cp gs://<bucket>/gradle.properties .
+```
+
+NOTE: If you would like to generate yourself: See template: [java/java-dataflow-samples/read-pubsub-write-bigquery/infrastructure/03.io
+/gradle.properties.tmpl](infrastructure/03.io/gradle.properties.tmpl).
+
+## Select the included IntelliJ Run Configuration
```

Comment:
should we mention a user should be logged in before running or an error message will state that clearly? We can say something like "Use Tools -> Google Cloud Code -> Sign in to Google Cloud Platform..."

Reply:
I'm putting Tools -> Google Cloud Code -> Sign in to Google Cloud Platform as the first requirement instead as well as repeated  under the Getting Started with IntelliJ section.